src/funcTest/groovy/info/solidsoft/gradle/pitest/functional/OverridePluginFunctionalSpec.groovy

```diff
@@ -1,80 +1,92 @@
+            apply plugin: 'info.solidsoft.pitest'
+            apply plugin: 'nebula-override'
+            group = 'gradle.pitest.test'
+
+            buildscript {
                 repositories { mavenCentral() }
-                dependencies { testImplementation 'junit:junit:4.11' }
-            """.stripIndent()
+                dependencies { classpath 'com.netflix.nebula:gradle-override-plugin:1.12.+' }
+            }
+            repositories { mavenCentral() }
+            dependencies { testImplementation 'junit:junit:4.11' }
+        """.stripIndent()
+
         and:
-            writeHelloWorld('gradle.pitest.test.hello')
+        writeHelloWorld('gradle.pitest.test.hello')
+
         when:
-            ExecutionResult result = runTasksSuccessfully('pitest', '-Doverride.pitest.reportDir=build/treports')
+        ExecutionResult result = runTasksSuccessfully('pitest', '-Doverride.pitest.reportDir=build/treports')
+
         then:
-            result.standardOutput.contains('Generated 1 mutations Killed 0 (0%)')
-            fileExists('build/treports')
+        result.standardOutput.contains('Generated 1 mutations Killed 0 (0%)')
+        fileExists('build/treports')
     }
 
-    @Issue("https://github.com/szpak/gradle-pitest-plugin/issues/139")
-    @PendingFeature(exceptions = GradleException, reason = "Not implemented yet due to Gradle limitations described in linked issue")
-    def "should allow to define features from command line and override those from configuration"() {
+    @Issue('https://github.com/szpak/gradle-pitest-plugin/issues/139')
+    @PendingFeature(exceptions = GradleException, reason = 'Not implemented yet due to Gradle limitations described in linked issue')
+    void "should allow to define features from command line and override those from configuration"() {
         given:
-            buildFile << """
-                ${getBasicGradlePitestConfig()}
-
-                pitest {
-                    failWhenNoMutations = false
-                    timestampedReports = true
-                }
-            """.stripIndent()
+        buildFile << """
+            ${getBasicGradlePitestConfig()}
+
+            pitest {
+                failWhenNoMutations = false
+                timestampedReports = true
+            }
+        """.stripIndent()
+
         when:
-            ExecutionResult result = runTasksSuccessfully('pitest', '--timestampedReports=false',
-                '--features=+EXPORT', '--features=-FINFINC')
+        ExecutionResult result = runTasksSuccessfully('pitest', '--timestampedReports=false',
+            '--features=+EXPORT', '--features=-FINFINC')
+
         then:
-            result.standardOutput.contains("--timestampedReports=false")
+        result.standardOutput.contains('--timestampedReports=false')
+
         and:
-            result.standardOutput.contains("--features=+EXPORT,-FINFINC")
+        result.standardOutput.contains('--features=+EXPORT,-FINFINC')
```

Comment:
I'm fine with `""`. It generates really low overhead for strings that do not contain `$`. I'm not sure I would like to change it everywhere.

Reply:
Will disable UnnecessaryGString rule